lib/audit/checksum.rb

```diff
@@ -50,4 +50,18 @@ def self.validate_disk_all_endpoints_profiled
     profiler.print_results_flat('CV_checksum_validation_all_endpoints')
   end
 
+  def self.validate_druid(druid)
+    start_msg = "#{Time.now.utc.iso8601} CV validate_druid starting for #{druid}"
+    puts start_msg
+    Rails.logger.info start_msg
+    po = PreservedObject.find_by(druid: druid)
+    pres_copy = PreservedCopy.find_by(preserved_object: po)
```

Comment:
one thing worth noting is that this assumes that there's only one pres copy per druid.  which is true for now, but won't be true once we have archive copies.

one option:  query like this:
```ruby
PreservedCopy.joins(:preserved_object).where(preserved_objects: {druid: druid})
```

and then loop over the results.

for now, the behavior would be equivalent.  and when we do have archive copies, we'd have to filter them out, or have `ChecsumValidator#validate_checksums` skip them or otherwise do something sensible.

re: the query we were just talking about F2F, where you needed to get PCs for a given druid, i think you could turn the `joins(:preserved_object).where(preserved_objects: {druid: druid})` part of the above AR query into a scope on `PreservedCopy`, and chain it with the other scopes you're using.

Reply:
See #263 re: one pres_copy re druid-endpoint